README.md

```diff
@@ -79,19 +79,19 @@ ninja install
 
 ## Using VKD3D
 
-VKD3D can be used by projects that target Direct3D 12 as a drop-in replacement
-at build-time with some modest source modifications.
+VKD3D can be used by projects targeting Direct3D 12 as an alternative
```

Comment:
It's a drop-in replacement as opposed to an alternative, especially in cases where there is nothing to use in the first place.